assets/src/domain/eventEditor/ui/ticketAssignmentsManager/components/ModalContainer.tsx

```diff
@@ -10,9 +10,9 @@ const ModalContainer: React.FC<ModalContainerProps> = ({ isOpen, onClose, ...pro
 
 	let title = '';
 	if (assignmentType === 'forDate') {
-		title = sprintf(__('Ticket Assignment Manager for Datetime: %s - %s'), entity.dbId, entity.name);
+		title = sprintf(__('Ticket Assignment Manager for Datetime: %s - %s'), String(entity.dbId), entity.name);
 	} else if (assignmentType === 'forTicket') {
-		title = sprintf(__('Ticket Assignment Manager for Ticket: %s - %s'), entity.dbId, entity.name);
+		title = sprintf(__('Ticket Assignment Manager for Ticket: %s - %s'), String(entity.dbId), entity.name);
```

Comment:
![image](https://user-images.githubusercontent.com/1477453/79576564-2d4e2080-80cc-11ea-90b7-132e9bd1b299.png)

Reply:
crazy... never had that before

Reply:
I had that many times 😄